### ISSUE TITLE
New RabbitMQ Image / Version 3.9.x

### ISSUE THREAD

**Issue by discostur** [bitnami/charts#9978]
### Name and Version

rabbitmq-cluster-operator-2.6.0

### What is the problem this feature will solve?

RabbitMQ Cluster Operator should use RabbitMQ version 3.9.x since version 12.x:

https://github.com/rabbitmq/cluster-operator/commit/284eed9b82783e165177b09d4caa7cc0a08555d5

Just installed rabbitmq-cluster-operator-2.6.0 via helm from bitnami and bootstrapped a cluster. The image used is still 3.8.x:

```
image: docker.io/bitnami/rabbitmq:3.8.29-debian-10-r7
``` 

I didn't set any specific image tag on cluster creation.

**Comment by carrodher** [comment]
Thanks for letting us know, I just created an internal task to modify our internal test & release pipeline

**Comment by carrodher** [comment]
We just released a new Helm chart version (2.6.1) using 3.9.x in the RabbitMQ container image, see https://github.com/bitnami/charts/commit/edd38adcc32cde000a530e3e5d998a86a9b03eed

```console
$ helm repo update bitnami
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈

$ helm search repo bitnami/rabbitmq-cluster-operator
NAME                             	CHART VERSION	APP VERSION	DESCRIPTION
bitnami/rabbitmq-cluster-operator	2.6.1        	1.12.1     	The RabbitMQ Cluster Kubernetes Operator automa...

$ helm fetch bitnami/rabbitmq-cluster-operator --untar
$ ag 'tag:' -B 1 rabbitmq-cluster-operator/values.yaml
63-  repository: bitnami/rabbitmq
64:  tag: 3.9.16-debian-10-r2
--
83-  repository: bitnami/rmq-default-credential-updater
84:  tag: 1.0.2-scratch-r2
--
105-    repository: bitnami/rabbitmq-cluster-operator
106:    tag: 1.12.1-scratch-r9
--
483-    repository: bitnami/rmq-messaging-topology-operator
484:    tag: 1.5.0-scratch-r2
```

**Comment by discostur** [comment]
@carrodher thanks for your fast help ;)
